net.efaxgtk.efaxgtk.yaml

```diff
@@ -0,0 +1,70 @@
+app-id: net.efaxgtk.efaxgtk
```

Comment:
efaxgtk.net doesn't exist.

Reply:
Ah, I didn't realize these represented actual hostnames :)

The website is at: http://efax-gtk.sourceforge.net/

net.sourceforge.efax-gtk?

Do I rename everything but keep using this pull request?

Reply:
yep.

Reply:
I used net.sourceforge.efaxgtk.  Let me know if there's a more appropriate name.